defaults/main.yml

```diff
@@ -101,6 +111,9 @@ omero_server_python_requirements:
   - numpy
   - pillow
   - tables
```

Comment:
tables is now included in omero-server.

Reply:
It's not in https://github.com/ome/omero-cli-server/blob/v0.1.0/setup.py#L51-L55

Reply:
true only if you use ``[default]``